apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -1022,8 +1061,10 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     * @param _voter Address of principal voter
     * @param _caster Address of vote caster, if voting via representative
     */
+    // Note: not sure about pushing these to be private 🤷‍♂️
```

Comment:
Wasn't really sure I understood what was meant to be private, but from the current ones, I'm not convinced any really need to be private 🤷

Reply:
My main intention was to make sure we don't expose unsafe or incomplete functions to inheriting contracts, but looking at the exposed interface, it doesn't seem harmful now. Addressed in 6e2ec983f8496dadbe110a88de7fdfe728465c6f